### ISSUE TITLE
07_NaturalLanguageProcessing_04_SkipGram: Suggestion for line 270 valid_word

### ISSUE THREAD

**Issue by ychong** [nfmcclure/tensorflow_cookbook#121]
Suggestion for a shorter implementation of line 270 for 'valid_word':

Old:
valid_word = word_dictionary_rev[valid_examples[j]]

Suggestion/New:
valid_word = valid_words[j]

**Comment by nfmcclure** [comment]
Thanks!  I'll update this code when I get to chapter 7 with all the new updates (version 2 of the book coming soon).

**Comment by nfmcclure** [comment]
I'm about to push a fix.  I have no idea why this is.  But if you lower the learning rate to 0.25 it seems to work.

Change:
`model_learning_rate = 0.25`

You may also consider adding:
`tf.set_random_seed(42)'
`np.random.seed(42)`

I got it to run that way.  Look for the code update later today.

**Comment by nfmcclure** [comment]
This should be fixed now.
